main.js

```diff
@@ -0,0 +1,11 @@
+
+  $.ajax({
+    url: "https://randomuser.me/api/?inc=name,email,id&results=10",
+    dataType: "json",
+    success: function (data) {   
+        const source = $("#people-template").html()
+        const template = Handlebars.compile(source)
+        const newHTML = template(data.results);
+        $('#container').append(newHTML);
```

Comment:
try to error handle. you have the `success` but there is also an opportunity to use the `error`